src/identity/components/internal/WithNameBadge.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Address } from 'viem';
```

Comment:
Avoid create a sub folder `internal`, let's keep the code as flat as possible. It will make it easier to mantain.